### PR TITLE
Add convenience dependency package pulling in openQA-worker

### PR DESCRIPTION
**Pull request by okurz**

### CODE REVIEW

File: os-autoinst-distri-opensuse-deps.spec [hunk 1]
```diff
@@ -81,6 +81,15 @@ Recommends:     os-autoinst-devel
 %description
 Metapackage that contains the dependencies of os-autoinst-distri-opensuse.
 
+%package worker
+Summary:        Convenience package pulling in os-autoinst-distri-openSUSE dependencies and the openQA worker
+Group:          Development/Tools/Other
+Requires:       %{name} = %{version}
+Requires:       openQA-worker
+
+%description no-selenium-devel
```
Comment (foursixnine) [inline os-autoinst-distri-opensuse-deps.spec:90]: I guess it's leftover from something?

```suggestion
%description
```

**Review by foursixnine — APPROVED** [review]

**Comment by foursixnine** [comment]
Thanks a lot @okurz

**Comment by okurz** [comment]
You want to mention it in documentation somewhere?

**Comment by foursixnine** [comment]
> You want to mention it in documentation somewhere?

Yes, for now: https://progress.opensuse.org/issues/123301 will take care of that, as I identified a new req.

**Comment by okurz** [comment]
My suggestion for followup: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/16260

**Pull request merged**